drivers/media/pci/intel/ipu-isys-video.c

```diff
@@ -38,6 +38,13 @@
 /* use max resolution pixel rate by default */
 #define DEFAULT_PIXEL_RATE	(360000000ULL * 2 * 4 / 10)
 
+#define MAX_VIDEO_DEVICES	8
+
+static int video_nr[MAX_VIDEO_DEVICES] = { [0 ...(MAX_VIDEO_DEVICES - 1)] = -1 };
```

Comment:
Why not add the module parameter as the default start video_nr (1)?

Reply:
@jwrdegoede 
I just noticed that v4l2loopback support multiple devices with `devices` and `video_nr` arguments. 
However, is it OK that IPU6 skip just `/dev/video0` for v4l2loopback on Linux distros? If it is not good enough for v4l2loopback,  it is fine for me to pick this change.

Reply:
@bingbucao 

Thank you for looking a this.

Right the video_nr module argument is modelled after v4l2loopback.

As for only skipping /dev/video0 some devices e.g. the Juno tablet https://junocomputers.com/us/product/juno-tablet/ combine a UVC camera on one side with an IPU6 camera on the other side.

So a system may have a mix of USB(UVC) and IPU6 cameras and then we really want to the not directly usable /dev/video# node(s) from IPU6  to be numbered after both the v4l2loopback node and the UVC camera nodes.

So I believe it is best to merge this pull-request as is and just let the distros deal with which numbers to pick by setting the parameter.

Reply:
Ack. Thanks.